### PR TITLE
Add support for positive, negative and warning buttons

### PR DESCRIPTION
**Pull request by Vineet119**

### CODE REVIEW

File: packages/theme/css/characteristics/actionable-next.css [hunk 1]
```diff
@@ -124,4 +124,130 @@
...
+  --salt-actionable-positive-bold-foreground-hover: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-positive-bold-foreground-active: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-positive-bold-foreground-disabled: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-positive-bold-borderColor: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-borderColor-hover: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-borderColor-active: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-borderColor-disabled: var(--salt-palette-alpha-none);
+  --salt-actionable-positive-bold-background: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-background-hover: var(--salt-palette-positive-action-hover);
+  --salt-actionable-positive-bold-background-active: var(--salt-palette-positive);
```
Comment (pseys) [inline packages/theme/css/characteristics/actionable-next.css:181]: The active state on the 'next' positive button background is incorrect. It should be `--salt-palette-positive-action-active`

Comment (joshwooding) [inline packages/theme/css/characteristics/actionable-next.css:181]:
```suggestion
  --salt-actionable-positive-bold-background-active: var(--salt-palette-positive-action-active);
```

**Comment by joshwooding** [comment]
Converting to draft whilst discussions are happening

**Comment by origami-z** [comment]
closing in favor of #4067

**Pull request closed without merging**